### DISCUSSION TITLE
Can't Login in Edge 110/Windows 10

### DISCUSSION THREAD

**Discussion by ymcheung** [community/community#48939] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello!
I use Github on macOS generally, and sometimes need to browse something on a Windows 10 PC. However, I got 404 when trying to login with Edge's auto-fill.

Step 1: Click `Sign in` at the top right corner of Github homepage
Step 2: Edge shows the auto-fill data (synced with macOS)
![螢幕截圖 2022-11-26 00 21 10](https://user-images.githubusercontent.com/4409977/222307224-d378cfd5-9333-4745-9e7a-116671fea28c.png)
Step 3: Got 404 and a pop-over modal with auto-fill data
![螢幕截圖 2022-11-26 00 21 17](https://user-images.githubusercontent.com/4409977/222307349-f56b4749-bbf1-4e1b-b37c-5d7da7e4ad30.png)

Try to login with that modal and got 404 again.

The auto-fill data work perfectly on macOS edge.

I think the console message won't help?
![螢幕截圖 2023-02-06 23 37 04](https://user-images.githubusercontent.com/4409977/222307867-19e40a17-a820-4ec6-9d11-3e41c3364253.png)

Thank you!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ymcheung ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by ghost** [comment]
Hi there! Sorry to hear you're having trouble logging into GitHub on Edge for Windows 10.

It's possible that the auto-fill data from macOS isn't compatible with Edge for Windows 10, or that there's a temporary issue with the login process. Here are a few steps you can try to troubleshoot the problem:

- Clear your Edge browser cache and cookies: This can sometimes fix login issues. To do this, go to the Edge settings menu, select "Privacy, search, and services," and then click "Choose what to clear" under the Clear browsing data section.

- Try manually typing in your login information: Instead of relying on auto-fill, try manually entering your username and password to see if that works.

- Check your network connection: Make sure your Windows 10 PC is connected to the internet and that there are no network issues that could be preventing you from logging in.

If none of these steps work, you could also try using a different browser on your Windows 10 PC, such as Chrome or Firefox, to see if that solves the issue.

I hope one of these solutions helps you log in successfully. Let me know if you have any further questions or concerns!
